### DISCUSSION TITLE
Considerations for integration with Raven Chat or the Matrix protocol

### DISCUSSION THREAD

**Discussion by blaggacao** [frappe/gameplan#183] [Ideas]
> We built it for ourselves because we were finding it difficult to keep track of so many conversations in our chat tool.

Me too, 100%.

But before I roll on, let me introduce the [matrix protocol](https://matrix.org/), in case it's not known (it probably is :stuck_out_tongue_winking_eye: ):

> An open network for secure, decentralized communication.

Where "decentralized", fortunately is pretty optional if you run your own so called "home server". Already hear where this is going?

Cause sometimes it's useful to have a synchronous escape hatch such as an ephimeral chat or a quivideo conference.

Matrix features the Element Chat App which not only has a pretty good integration with jitsii (for call & video conferences) but furthermore implements so called "spaces" which is an astonishingly precise mapping of Gameplan's Teams.

So let's put this prose introduction into some order...
---

# User Story

As a users of gameplan, we want to have an escape hatch into synchronous communication to which we count traditional chat as gameplan's readme explains.

We want to be able to convene a quick back and forth on urgent or rather transactional matters via chat or video with the ability to `/record` a summary back into game plan.

We want game plan's teams to automatically be reflected as matrix "spaces" with the same access rights as gameplan.

---

I may come back with further context after I explore this more. But if you want to understand from where I'm coming from, maybe setting up Element Chat and play with the spaces feature might be the best bet.

**Comment by netchampfaris** [comment]
@blaggacao 

Hey, we ourselves use Telegram for synchronous/quick chat. So chat is not completely avoidable.
I am [exploring](https://github.com/frappe/gameplan/pull/345) a new direction for Gameplan. I have plans to add a chat feature too.

I don't have a lot of context about Matrix, but i'll check it out.

edit: corrected link

**Reply by blaggacao** [reply]
Hey, in the meantime, we've actually abandoned Matrix Chat ourselves in favor of the relatively new Raven Chat due to its better integration into Frappe.

So the same might make sense to us, but for Raven.

Btw, the link you shared just points to this discussion. This might have been a typing mistake.

**Comment by MartinFulgueiras** [comment]
Hi Guys/Gals, 

Any news or link about where gameplan is heading with the announcement of Raven being taken into the frappe team? 
They seem to be doing something similar... or am I mistaken? If this has already been discussed please guide me to the link. 

Thanks

**Reply by MartinFulgueiras** [reply]
Mybe this is the answer although I need to get more into it: 
https://discuss.frappe.io/t/whats-the-difference-between-gameplan-and-raven/132201
